.gitpod.yml

```diff
@@ -0,0 +1,9 @@
+# This configuration file was automatically generated by Gitpod.
+# Please adjust to your needs (see https://www.gitpod.io/docs/config-gitpod-file)
+# and commit this file to your remote git repository to share the goodness with others.
+
+tasks:
+  - init: npm install && npm run build
+    command: npm run start
+
+
```

Comment:
You'll need to remove this file from the commit, this is a configuration file for your GitPod it doesn't need to be in main codebase